esx_service/utils/convert.py

```diff
@@ -24,7 +24,11 @@ def convert_to_MB(vol_size_str):
     """
 
     unit = vol_size_str[-2:].upper()
-    value = int(vol_size_str[:-2])
+    try:
```

Comment:
Isn't this function supposed to return result once its set?

def check_privileges_for_command(cmd, opts, tenant_uuid, datastore_url, privileges):
    """
        Check whether the (tenant_uuid, datastore) has the privileges to run
        the given command.

    """
    result = None
    if not privileges:
        result = error_code_to_message[ErrorCode.PRIVILEGE_NO_PRIVILEGE]
        return result

    cmd_need_mount_privilege = [CMD_ATTACH, CMD_DETACH]
    if cmd in cmd_need_mount_privilege:
  ......

    if cmd == CMD_CREATE:
        if not has_privilege(privileges, auth_data_const.COL_ALLOW_CREATE):
            result = error_code_to_message[ErrorCode.PRIVILEGE_NO_CREATE_PRIVILEGE] <---
        if not check_max_volume_size(opts, privileges):
            result = error_code_to_message[ErrorCode.PRIVILEGE_MAX_VOL_EXCEED] <----
        if not check_usage_quota(opts, tenant_uuid, datastore_url, privileges):
            result = error_code_to_message[ErrorCode.PRIVILEGE_USAGE_QUOTA_EXCEED] <----

    if cmd == CMD_REMOVE:
      .....

    return result